x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/fields_panel/fields_panel.tsx

```diff
@@ -78,51 +79,51 @@ export const FieldsPanel: FC<Props> = ({
   }
 
   return (
-    <EuiPanel data-test-subj={`mlDataVisualizerFieldsPanel ${fieldTypes}`}>
-      <EuiTitle>
-        <h2>{title}</h2>
-      </EuiTitle>
-      <EuiSpacer size="s" />
-      <EuiFlexGroup alignItems="center">
-        <EuiFlexItem grow={false}>
-          <span>
-            {showAllFields === true ? (
-              <FormattedMessage
-                id="xpack.ml.datavisualizer.fieldsPanel.showAllCountDescription"
-                defaultMessage="{wrappedCardsCount} {cardsCount, plural, one {field} other {fields}} ({wrappedPopulatedFieldCount} {populatedFieldCount, plural, one {exists} other {exist}} in documents)"
-                values={{
-                  cardsCount: fieldVisConfigs.length,
-                  wrappedCardsCount: <b>{fieldVisConfigs.length}</b>,
-                  populatedFieldCount,
-                  wrappedPopulatedFieldCount: <b>{populatedFieldCount}</b>,
-                }}
-              />
-            ) : (
-              <FormattedMessage
-                id="xpack.ml.datavisualizer.fieldsPanel.fieldsCountDescription"
-                defaultMessage="{wrappedCardsCount} {cardsCount, plural, one {field exists} other {fields exist}} in documents ({wrappedTotalFieldCount} in total)"
-                values={{
-                  cardsCount: fieldVisConfigs.length,
-                  wrappedCardsCount: <b>{fieldVisConfigs.length}</b>,
-                  wrappedTotalFieldCount: <b>{totalFieldCount}</b>,
-                }}
-              />
+    <div data-test-subj={`mlDataVisualizerFieldsPanel ${fieldTypes}`}>
+      <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+        <EuiFlexItem>
+          <EuiFlexGroup alignItems="center" gutterSize="m">
+            <EuiFlexItem grow={false}>
+              <EuiTitle size="m">
+                <h2>{title}</h2>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiBadge>
+                <b>{fieldVisConfigs.length}</b>
+              </EuiBadge>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
```

Comment:
The numbers in this section aren't correct.

e.g. for cloudwatch, 5000 docs per shard:

![image](https://user-images.githubusercontent.com/7405507/72253064-93706680-35f8-11ea-940a-8a72534a1bda.png)

The numbers should be:

![image](https://user-images.githubusercontent.com/7405507/72253115-ae42db00-35f8-11ea-9272-f69ce64f5ed7.png)

The first number (now in the grey square) correctly shows the number of fields that appear in the documents sampled. The second, total, number, should the total number of fields that are in the index mapping. Here it always seems to be equal to the first number.

Reply:
Fixed in https://github.com/elastic/kibana/pull/54358/commits/5a39b629f5283016edd7dbfeaadd57ba09dc54f8

Reply:
These numbers are the wrong way round. Total fields should read `8`, with `7` appearing in documents sampled:

![image](https://user-images.githubusercontent.com/7405507/72341821-237ae280-36c3-11ea-8b30-c890a9a7650c.png)

Reply:
https://github.com/elastic/kibana/pull/54358/commits/c16861c376ac8baff8989dcb59522ac8f09680a0